src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public Ball getRandomBall() {
+        int randomNumber = new Random().nextInt(100);
```

Comment:
According to our checklist, all magic numbers in your code should be constants. Please, move `100` to a constant field. In addition, creating a new Random object every time you need a random number is inefficient. It would be better to create a single Random instance as a private field in your class and reuse it.